docs/intro.asciidoc

```diff
@@ -26,6 +26,37 @@ by instrumenting both XHR and Fetch API requests from the webpage to backend ser
 
 For all transactions, the agent automatically captures <<breakdown-metrics-docs, breakdown metrics>>.
 
+[float]
+[[page-load-metrics]]
+==== Page load metrics
+
+The RUM agent tries to capture the overall user experience on how the page is loaded/rendered as part of the page load Transaction.
+It includes all the dependent resources such as JavaSript, Stylesheets, Images, etc. which are included as part of the page. In addition
+to capturing the resource timing information as Spans, the transaction also includes navigation spans that are associated along with the rest
+of the captured information to make the waterfall more appealing.
+
+* `Domain lookup` - Duration of the DNS query for the current page (domainLookupEnd - domainLookupStart).
+
+* `Making a connection to the server` - Duration of TCP connection to the server. Includes the TLS negotiation time for HTTPS pages (connectEnd - connectStart).
+
+* `Requesting and receiving the document` - Overall response time of the server including the last byte (responseEnd - requestStart).
+
+* `Parsing the document, executing sync. scripts` - HTML document parsing time including synchronous Stylesheets and Script tags (domInteractive - domLoading).
+
+* `Fire "DOMContentLoaded" event` - Triggered when the browser completed parsing the document. Helpful when there are multiple listeners or logic
+is executed (domContentLoadedEventEnd - domContentLoadedEventStart)
+
+* `Fire "load" event` - Trigged when the browser finishes loading its document and depedent resources (loadEventEnd - loadEventStart).
```

Comment:
What do you think about reformatting this? I find this easier to read:
<img width="666" alt="Screen Shot 2019-12-19 at 12 54 04 PM" src="https://user-images.githubusercontent.com/5618806/71208797-b9575500-225e-11ea-8699-c1365711519b.png">

If you like it too, here's the copy/pasta:
```asciidoc
`Domain lookup`::
Duration of the DNS query for the current page (domainLookupEnd - domainLookupStart).

`Making a connection to the server`::
Duration of TCP connection to the server. Includes the TLS negotiation time for HTTPS pages (connectEnd - connectStart).

`Requesting and receiving the document`::
Overall response time of the server including the last byte (responseEnd - requestStart).

`Parsing the document, executing sync. scripts`::
HTML document parsing time including synchronous Stylesheets and Script tags (domInteractive - domLoading).

`Fire "DOMContentLoaded" event`::
Triggered when the browser completed parsing the document. Helpful when there are multiple listeners or logic
is executed (domContentLoadedEventEnd - domContentLoadedEventStart)

`Fire "load" event`::
Trigged when the browser finishes loading its document and depedent resources (loadEventEnd - loadEventStart).
```

Reply:
Yeah it looks nicer, I will apply your suggestions.